src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star star-active"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
```

Comment:
```suggestion
    <div class="stars stars--1">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
Stars elements should have any additional classes. Use `stars--N` modifier to set the active stars
![image](https://github.com/mate-academy/layout_stars/assets/96409563/f8b185e8-4caf-40e5-9c9f-03b5cf10ad48)